### PR TITLE
Typo fixes for classes-emit.md

### PR DESCRIPTION
**Pull request by Marketionist**
**Added typo fixes for classes-emit.md**

Also:
- added .DS_Store to .gitignore

### CODE REVIEW

**Comment by Marketionist** [comment]
@basarat please review and merge if it's ok

**Comment by basarat** [comment]
I've merged and will revert what I mentioned myself :rose:

File: docs/classes-emit.md [hunk 1]
```diff
@@ -54,15 +54,15 @@ Here `d` refers to the derived class and `b` refers to the base class. This func
...
 #### `d.prototype.__proto__ = b.prototype`
 
 After having tutored many people about this I find the following explanation to be simplest. First we will explain how the code from `__extends` is equivalent to the simple `d.prototype.__proto__ = b.prototype`, and then why this line in itself is significant. To understand all this you need to know these things:
 1. `__proto__`
-1. `prototype`
-1. effect of `new` on `this` inside the called function
-1. effect of `new` on `prototype` and `__proto__`
+2. `prototype`
+3. effect of `new` on `this` inside the called function
+4. effect of `new` on `prototype` and `__proto__`
```
Comment (Marketionist) [inline docs/classes-emit.md]: Yes, numbers get properly updated on the website (https://basarat.gitbooks.io/typescript/content/docs/classes-emit.html) and in .pdf version, but looks like github preview doesn't update numbers - please see the screenshot

numbered-list
(https://github.com/basarat/typescript-book/blob/master/docs/classes-emit.md). So, explicit numbering with 1,2,3,4 can be more universal solution. What do you think about it @basarat ?

Comment (basarat) [inline docs/classes-emit.md]: Just needed an enter. Fixed https://github.com/basarat/typescript-book/commit/237c9fb36dd395c7d762dbbef03a37e14424137a :rose:

**Pull request merged**